etc/Dockerfile.mod

```diff
@@ -1,7 +1,6 @@
 ARG BASE_TAG
```

Comment:
I know this file is not new, but would add comment here explaining what this is for + how it differs from the base image

Reply:
Sure thing - included an overview of the docker stages in `doc/DEVELOP.md`.